Api/AdministratorServices/AgencySystemSettingsManagementService.cs

```
@@ -74,8 +74,12 @@ AgencyAccommodationBookingSettings GetDefaults()
                 {
                     IsSupplierVisible = false,
                     IsDirectContractFlagVisible = false,
-                    AprMode = AprMode.Hide,
-                    PassedDeadlineOffersMode = PassedDeadlineOffersMode.Hide,
+                    AprMode = agency.ContractKind is ContractKind.OfflineOrCreditCardPayments 
```

If there are no settings, then we give the default always Hide